### ISSUE TITLE
Ignore global vars prefixed with `{E,e}rr`

### ISSUE THREAD

**Issue by leonklingele** [leighmcculloch/gochecknoglobals#2]
```
x.go:11::warning: ErrHandlerAlreadyExists is a global variable (gochecknoglobals)
```

It's a common practice to create global error variables such as the one above. It might be wise to not warn about them. What do you think?

**Comment by leighmcculloch** [comment]
Yeah that's a good point. We could ignore global variables starting with `Err` and that satisfy the `error` Interface.

**Comment by leighmcculloch** [comment]
Would you be up for taking a stab at this and opening a PR?

**Comment by leonklingele** [comment]
Sure! Will take a look at it later today.

**Comment by leonklingele** [comment]
For the records: This has been implemented in #3. Don't know why GH doesn't crosslink to it.

**Comment by leighmcculloch** [comment]
You need to reference this issue in the PR description for GH to crosslink it. Unfortunately the reference in the PR title doesn't crosslink. I just added the `Fixes #2` to the PR description and it has linked it now.
